metron-deployment/roles/elasticsearch/tasks/create-indices.yml

```diff
@@ -32,7 +35,6 @@
   until: result.content.find("green") != -1
   retries: 10
   delay: 60
-  run_once: yes
 
```

Comment:
Because we want everything in `create-indices.yml` to "run once".  I pulled `run_once: yes` out to `roles/elasticsearch/tasks/main.yml`. Make sense?

```
- include: create-indices.yml
  run_once: yes
```